<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>image/login.png</normaloff>image/login.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
      color: rgb(74, 74, 111);
	  background-color: rgb(168, 218, 220);
    
}
QPushButton {
 font: bold;
 font-size:12px;
 line-height:1.2;
  color: #f8f9fa;
  justify-content: center;
  align-items: center;
  padding: 0 20px;
  width: 100%;
  height: 50px;
  background-color: rgb(80, 80, 150);
  border-radius: 10px;

	
}

QPushButton:hover {
  background-color: rgb(74, 74, 125);
  box-shadow: 0 10px 30px 0px rgba(51, 51, 51, 0.5);
 
}

QPushButton:pressed {
   background-color: rgb(74, 74, 111);
}</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="user_text">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Poppins-Medium,sans-serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-image: url(img/Standar/user.png);
    background-repeat: no-repeat;
    background-position: left;
    color: rgb(74, 74, 111);
	font-family:'Poppins-Medium', sans-serif;
    font-size: 12px;
    padding: 4 4 4 35; /* left padding (last number) must be more than the icon's width */
	border-radius:10px;
	border-bottom:2px solid #d9d9d9;
	background-color: rgb(241, 250, 238);
}

QLineEdit:hover{
    border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
    border: 2px solid rgb(85, 170, 255);
    background-color: rgb(241, 250, 238);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string> User Name</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_text">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins-Medium,sans-serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-image: url(img/Standar/password.png);
    background-repeat: no-repeat;
    background-position: left;
   color: rgb(74, 74, 111);
	font-family:'Poppins-Medium', sans-serif;
    font-size: 12px;
    padding: 4 4 4 35; /* left padding (last number) must be more than the icon's width */
	border-radius: 10px;
	border-bottom:2px solid #d9d9d9;
	background-color: rgb(241, 250, 238);

}

QLineEdit:hover{
    border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
    border: 2px solid rgb(85, 170, 255);
    background-color: rgb(241, 250, 238);
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string> Password</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Login_But">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelaccount">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>320</width>
      <height>390</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(img/logo_mood.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"> font-size:25px;
 line-height:1.2;
  color: rgb(74, 74, 111);
  justify-content: center;
  align-items: center;
  padding: 0 20px;
  width: 100%;
  height: 50px;
</string>
    </property>
    <property name="text">
     <string>Mood Prediction
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signin_But">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins-Medium,sans-serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(222, 62, 8);
font-family:'Poppins-Medium', sans-serif;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string>Oops! ,Username Not Found.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(222, 62, 8);
font-family:'Poppins-Medium', sans-serif;
font-size: 13px;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
